src/runtime-rs/crates/runtimes/src/shim_mgmt/client.rs

```diff
@@ -74,13 +70,19 @@ impl MgmtClient {
             .uri(url)
             .header("content-type", content_type)
             .body(body)?;
-        let work = self.client.request(req);
+        return self.send_request(req).await;
+    }
+
+    async fn send_request(&self, req: Request<Body>) -> Result<Response<Body>> {
+        let msg: String = format!("Request ({:?}) to uri {:?}", req.method(), req.uri());
```

Comment:
@jodh-intel Hi James, thanks for the review, it really helped ironing out the details. I have one minor question with how logging in Rust should work tho. I couldn't allocate error messages with request info after the request was sent due to the borrowing mechanism in Rust, so I had to pre-build the logging message in line 77. This is more or less cumbersome since most of the time the request would take the OK branch, could you perhaps share some advice on this?😀